src/sentry/search/events/filter.py

```diff
@@ -1131,7 +1131,27 @@ def _default_filter_converter(self, search_filter: SearchFilter) -> Optional[Whe
                 # If not a tag, we can just check that the column is null.
                 return Condition(Function("isNull", [lhs]), Op(search_filter.operator), 1)
 
-        if search_filter.value.is_wildcard():
+        if name in ARRAY_FIELDS and search_filter.value.is_wildcard():
+            condition = Condition(
```

Comment:
nit: could we restructure these ifs a bit so all the array stuff is next to each other? eg.
```suggestion
        if name in ARRAY_FIELDS:
            if search_filter.value.is_wildcard():
                condition = Condition(
```